src/asset-relocator.js

```diff
@@ -307,7 +335,7 @@ module.exports = async function (content, map) {
   // injection to set __webpack_require__.ab
   const options = getOptions(this);
 
-  injectPathHook(this._compilation, options.outputAssetBase);
+  injectPathHook(this._compilation, options.outputAssetBase, options.relativeToSource);
```

Comment:
I'm not sure I understand the behavior for this option -- if you set `relativeToSource: true` it seems to rewrite both reads *and* writes to be relative to source? This almost seems like it'd never be what you want.

Wouldn't typical behavior be to want to rewrite reads but leave writes in their normal location.

Also, can you please point me to a library that needs write support? I've brought it up a couple times, but I'd rather land this in Next.js without write support first.